source/appModules/calculator.py

```diff
@@ -66,28 +66,25 @@ def event_nameChange(self, obj, nextHandler):
 		nextHandler()
 
 	def event_UIA_notification(self, obj, nextHandler, displayString=None, activityId=None, **kwargs):
-		# #12268: for "DisplayUpdated", announce display strings in braille and move on.
+		calculatorVersion = int(self.productVersion.split(".")[0])
+		# #12268: for "DisplayUpdated", announce display strings in braille  no matter what they are.
 		if activityId == "DisplayUpdated":
 			braille.handler.message(displayString)
-		try:
-			shouldAnnounceNotification = (
-				obj.previous.UIAAutomationId in
-				("numberPad", "UnitConverterRootGrid")
-			)
-		except AttributeError:
```

Comment:
Removing this effectively changes the scope of the subsequent code, is the indent level as expected?